### ISSUE TITLE
Missing property commit.zookeeper.connect when running uReplicator

### ISSUE THREAD

**Issue by daluu** [uber/uReplicator#183]
Was running uReplicator today from docker image built using latest master code, using the example 1 setup (but also adding `--dstzk.config` to the worker script command).

Running uReplicator, get this error below, previously using older code and not using with `--dstzk.config`, things worked fine.

```
...
[2018-11-28 02:46:02,915] INFO Property zookeeper.connect is overridden to some-host:2181 (kafka.utils.VerifiableProperties:70)
...
Exception in thread "main" java.lang.IllegalArgumentException: requirement failed: Missing required property 'commit.zookeeper.connect'
	at scala.Predef$.require(Predef.scala:224)
	at kafka.utils.VerifiableProperties.getString(VerifiableProperties.scala:178)
	at kafka.mirrormaker.KafkaConnector.<init>(KafkaConnector.scala:40)
	at kafka.mirrormaker.WorkerInstance.start(WorkerInstance.scala:239)
	at kafka.mirrormaker.MirrorMakerWorker.main(MirrorMakerWorker.scala:109)
	at com.uber.stream.kafka.mirrormaker.starter.MirrorMakerStarter.main(MirrorMakerStarter.java:42)
```

I'm not aware that I needed to define a `commit.zookeeper.connect` property. What value (type) should be assigned to it, and where does this property go under? The property file referenced by `--dstzk.config` or the (default) consumer.properties file under uReplicator configs?

**Comment by xhl1988** [comment]
We are testing the fix: #186

**Comment by yangy0000** [comment]
fixed
